DEVELOPER_GUIDE.md

```diff
@@ -165,7 +165,16 @@ Run tests :
 ./gradlew clean build test
 ```
 
-## Run Tar file
+## Generate Artifact
+
+In opensearch-sdk navigate /build and look for distributions folder. If it is present than check for tar file in it. If it is not present then follow below step.
```

Comment:
I think it would be better to provide the path. i.e. `build/distributions/opensearch-sdk-1.0.0-SNAPSHOT.tar`